### PR TITLE
Config

### LINKED ISSUE #64: Separate out configgy stuff from data-y stuff in lists.json
Two separate files, so that lists.json can be generated without knowing desired config?

### PR DESCRIPTION
**Pull request by richardgoater**
No rush on the review for this, I know it's a big change.

I haven't gone for any back compat or any merging of old config, but I suppose this would be possible if you feel it's needed. There are also probably a few props that could be made optional by smart defaults, like any of the `default_*` properties, but they're required for now.

fixes #64.

### CODE REVIEW

**Comment by richardgoater** [comment]
Sorry I don't have access to the deployment :/

**Comment by theosanderson** [comment]
```


<!--StartFragment-->
19:55:23.684 | Could not resolve './CovInce' from src/App.jsx
-- | --
19:55:23.686 | error during build:
19:55:23.686 | Error: Could not resolve './CovInce' from src/App.jsx
19:55:23.686 | at error (/vercel/path0/node_modules/rollup/dist/shared/rollup.js:5279:30)
19:55:23.686 | at ModuleLoader.handleResolveId (/vercel/path0/node_modules/rollup/dist/shared/rollup.js:18717:24)
19:55:23.686 | at /vercel/path0/node_modules/rollup/dist/shared/rollup.js:18686:22
19:55:23.686 | at processTicksAndRejections (internal/process/task_queues.js:93:5)
19:55:23.686 | at async Promise.all (index 4)
19:55:23.686 | at async ModuleLoader.fetchStaticDependencies (/vercel/path0/node_modules/rollup/dist/shared/rollup.js:18684:34)
19:55:23.686 | at async Promise.all (index 0)
19:55:23.686 | at async ModuleLoader.fetchModule (/vercel/path0/node_modules/rollup/dist/shared/rollup.js:18661:9)
19:55:23.686 | at async Promise.all (index 4)
19:55:23.686 | at async ModuleLoader.fetchStaticDependencies (/vercel/path0/node_modules/rollup/dist/shared/rollup.js:18684:34)
19:55:23.701 | Error: Command "npx vite build" exited with 1

<!--EndFragment-->

```

let me know if you think it's at this end?

**Comment by theosanderson** [comment]
Wow - ton of awesome docs work

**Comment by theosanderson** [comment]
Looks like the file needs renaming or import changing?

**Comment by theosanderson** [comment]
Ah, linux is case sensitive and macOS is not :)

**Comment by theosanderson** [comment]
It doesn't really make sense (my bad) that `subnoun_plural` (etc.) are under `overview`. They should be under something like `area_naming` or something and maybe `heading` etc. should become `overview_heading` ?

File: README.md [hunk 1]
```diff
@@ -6,10 +6,80 @@ If you would like to deploy your own version of covince and need assistance plea
...
+
+* `domain [number[]]` (optional) [reference](https://recharts.org/en-US/api/YAxis#domain)
+* `ticks [string[]]` (optional) [reference](https://recharts.org/en-US/api/YAxis#ticks) 
+* `reference_line [number]` (optional) display a dotted line at the specified value
+* `allow_data_overflow [boolean]` (optional) [reference](https://recharts.org/en-US/api/YAxis#allowDataOverflow)
+
+### Map settings
+
+* `default_lineage [string]` should match a lineage in lists
+* `default_color_by [string]` should match a color in config
```
Comment (theosanderson) [inline README.md:42]: This doesn't need to match a color but a parameter

File: README.md [hunk 2]
```diff
@@ -6,10 +6,80 @@ If you would like to deploy your own version of covince and need assistance plea
...
+* `default_color_scale [string|object]` can be a string for all parameters, or an object for specific parameters. Omit parameters from the object to hide the scale control.
```
Comment (theosanderson) [inline README.md:43]: specify that it takes values 'linear' / 'quadratic'?

**Comment by theosanderson** [comment]
Super cool! Big improvement. Some brief comments above

File: README.md (hunk 2 above)
Comment (richardgoater) [reply README.md:43]: nice suggestion. Technically `R_scale` is an option, but we should generalise providing colour scales at some point. I'll make an issue for that.

File: README.md (hunk 1 above)
Comment (richardgoater) [reply README.md:42]: great spot, thanks 👍🏽

**Review by theosanderson — APPROVED** [review]
Looks great!!

**Pull request merged**